### PR TITLE
Add opcache page

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Inspector/Controller/OpcacheController.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Api\Inspector\Controller;
+
+use Psr\Http\Message\ResponseInterface;
+use Yiisoft\DataResponse\DataResponseFactoryInterface;
+
+use function Safe\opcache_get_status;
```
Comment (rustamwin) [inline src/Inspector/Controller/OpcacheController.php:10]: Is this a new dependency?

**Review by samdark — CHANGES_REQUESTED** [review]
See @rustamwin comment.

File: src/Inspector/Controller/OpcacheController.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{
+    public function __construct(
+        private DataResponseFactoryInterface $responseFactory,
+    ) {
+    }
+
+    public function index(): ResponseInterface
+    {
+        return $this->responseFactory->createResponse([
+            'status' => opcache_get_status(true),
```
Comment (vjik) [inline src/Inspector/Controller/OpcacheController.php:22]: Need to verify that opcache is available.

**Review by vjik — CHANGES_REQUESTED** [review]

File: src/Inspector/Controller/OpcacheController.php (hunk 1 above)
Comment (samdark) [reply src/Inspector/Controller/OpcacheController.php:10]: I guess, it was meant to be just `opcache_get_status` from root namespace. Right?

File: src/Inspector/Controller/OpcacheController.php (hunk 2 above)
Comment (samdark) [reply src/Inspector/Controller/OpcacheController.php:22]: Good idea. Or else it would just crash.

**Pull request merged**